lib/galaxy/security/idencoding.py

```diff
@@ -106,7 +106,10 @@ def decode_guid(self, session_key):
         # Session keys are strings
```

Comment:
Maybe a good time to add type annotation to this method?

```python
    def decode_guid(self, session_key: Union[bytes, str]) -> str:

Reply:
With type overloads it's not actually that bad, I'll push a commit